controller/search_blackbox_test.go

```diff
@@ -930,3 +930,83 @@ func (s *searchBlackBoxTest) TestIncludedParents() {
 	}
 	assert.Equal(s.T(), successCnt, 4)
 }
+
+func (s *searchBlackBoxTest) TestFilterAssigneeNullAfterWIUpdate() {
+	fxt := tf.NewTestFixture(s.T(), s.DB, tf.CreateWorkItemEnvironment(), tf.WorkItems(1))
+	spaceInstance := fxt.Spaces[0]
+	spaceIDStr := spaceInstance.ID.String()
+	filter := fmt.Sprintf(`
+							{"assignee":null}`,
+	)
+	_, result := test.ShowSearchOK(s.T(), nil, nil, s.controller, &filter, nil, nil, nil, nil, &spaceIDStr)
+	require.NotEmpty(s.T(), result.Data)
+	assert.Len(s.T(), result.Data, 1)
```

Comment:
Maybe you should add `assert.Nil(s.T(), result.Data[0].Attributes[workitem.SystemAssignees])` here to assert that `assignee` is `Nil` before workitem update as well

Reply:
Fixed. c808c7d963ad45ed719ee7cb9dc7a871ca1de8ec